### ISSUE TITLE
Pretraining PPO using off-policy data

### ISSUE THREAD

**Issue by donamin** [openai/baselines#967]
Hi

I want to pretrain the PPO policy mean and variance using off-policy data.
I defined a new training operation for the following loss:
`pretrain_loss = pg_loss - entropy * ent_coef`
(I removed the value loss because I only care about the policy network in the pretraining phase)

Then I wrote a code like this before the actual training:

`states, actions = load_off_policy_data()
nbatch = states.shape[0]
neglogpacs = np.zeros(states.shape[0])
for start in range(0, nbatch, nbatch_train):
	end = start + nbatch_train
	neglogpacs[start:end] = model.sess.run(model.neglogpac, {model.train_model.X: states[start:end, :], model.A: targets[start:end, :]})
inds = np.arange(nbatch)
for _ in range(noptepochs + 1):
	# Randomize the indexes
	np.random.shuffle(inds)
	# 0 to batch_size with batch_train_size step
	for start in range(0, nbatch, nbatch_train):
		end = start + nbatch_train
		mbinds = inds[start:end]
		slices = (arr[mbinds] for arr in (states, targets, neglogpacs))
		losses = model.pretrain(1e-4, 1, *slices)
`

Is this the correct way to do this?
